docs/integration/KafkaIntegration.md

```diff
@@ -20,12 +20,30 @@ Kafka topics are native streaming data input to Nussknacker and the native outpu
 
 ## Schema Registry integration
 
-Nussknacker integrates with Schema Registry. It is the source of topics available to choose from in Kafka sources and sinks. It also allows Nussknacker to provide syntax suggestions and validation. Nussknacker assumes that for the topic `topic-name` a schema `topic-name-value` and optionally `topic-name-key` (for the Kafka topic key) will be defined in the schema registry.
-
-Schemas are stored and managed by Confluent Schema Registry; it is [bundled with Nussknacker](/about/TypicalImplementation) in all deployment versions. Schemas can be registered in Schema Registry by means of REST API based CLI or using AKHQ, an open source GUI for Apache Kafka and Confluent Schema Registry. AKHQ is bundled with Nussknacker in all deployment versions.
-
-Nussknacker supports both JSON and Avro schemas, and JSON and Avro topic payloads. Detailed information on how Avro data should be serialized/deserialized can be found in [Confluent Wire Documentation](https://docs.confluent.io/platform/current/schema-registry/serdes-develop/index.html#wire-format).
-
+Nussknacker integrates with Schema Registries. It is the source of knowledge about:
+- Which topics are available in Kafka sources and sinks
+- What schema versions are available for topic
+- How code completions and validations on types described by schemas should work
+- How messages will be deserialized and serialized
```

Comment:
for me `serialized and deserialized ` sounds better, but it's a trifle, we can leave it as it is now.